### ISSUE TITLE
Camera.getPhoto incorrect menu/prompt position in Android landscape

### ISSUE THREAD

**Issue by AntoniGadea** [ionic-team/capacitor-plugins#1643]
### Plugin(s)

@capacitor/camera": "^5.0.0

### Capacitor Version
5.0.4

### Platform(s)
- Android SDK 33

### Current Behavior
When device is in landscape mode, Camera.getPhoto() prompt/menu  options is not correctly positioned .
Part of the menu overflows the screen in the bottom.
[https://freeimage.host/i/H4AOmaR](https://freeimage.host/i/H4AOmaR)
[https://freeimage.host/i/H4Aeo92](https://freeimage.host/i/H4Aeo92)

You can swipe up the menu to put it in the correct position:
https://freeimage.host/i/captura-de-pantalla-2023-06-06-las-172808.H4AeK6G

### Expected Behavior
The menu should appear in the correct position at the beginning without need of swipe up, like in portrait mode
[https://freeimage.host/i/H4AkqWF](https://freeimage.host/i/H4AkqWF)

### Code Reproduction

```
  const image: Photo = await Camera.getPhoto({
            allowEditing: false,
            quality: 90,
            resultType: CameraResultType.Base64,
            promptLabelHeader: this.libraryService.getLabel('lbl_photo'),
            promptLabelCancel: this.libraryService.getLabel('lbl_cancel'),
            promptLabelPhoto: this.libraryService.getLabel('lbl_get_from_gallery'),
            promptLabelPicture: this.libraryService.getLabel('lbl_take_photo')
        });
```

### Additional Context

```
const config: CapacitorConfig = {
  appId: xxxxxxxx',
  appName: 'xxxxxxx',
  webDir: 'www',
  bundledWebRuntime: false,
  appendUserAgent: "xxxxxx/2.10.0",
  plugins: {
    Keyboard: {
      resize: KeyboardResize.Ionic
    }
  }
};
```

```
{
  "name": "xxxxxx",
  "version": "1.0.0",
  "author": "xxxx",
  "homepage": "xxxxxxx/",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "^14.0.0",
    "@angular/core": "^14.0.0",
    "@angular/forms": "^16.0.4",
    "@angular/platform-browser": "^14.0.0",
    "@angular/platform-browser-dynamic": "^16.0.4",
    "@angular/router": "^14.0.0",
    "@capacitor/android": "^5.0.0",
    "@capacitor/app": "^5.0.0",
    "@capacitor/browser": "^5.0.0",
    "@capacitor/camera": "^5.0.0",
    "@capacitor/core": "^5.0.0",
    "@capacitor/haptics": "^5.0.0",
    "@capacitor/ios": "^5.0.0",
    "@capacitor/keyboard": "^5.0.0",
    "@capacitor/preferences": "^5.0.0",
    "@capacitor/splash-screen": "^5.0.0",
    "@capacitor/status-bar": "^5.0.0",
    "@datorama/akita": "^7.1.1",
    "@datorama/akita-ngdevtools": "^7.0.0",
    "@flowww/design-system": "^0.27.3",
    "@flowww/signaturepad": "^4.0.2",
    "@ionic/angular": "^6.1.9",
    "@ngx-translate/core": "^14.0.0",
    "@ngx-translate/http-loader": "^7.0.0",
    "color": "^4.2.3",
    "crypto-js": "^4.1.1",
    "ionicons": "^6.0.3",
    "lodash.deburr": "^4.1.0",
    "moment": "^2.29.4",
    "ng2-pdfjs-viewer": "^14.0.0",
    "rxjs": "~6.6.0",
    "timers": "^0.1.1",
    "tslib": "^2.2.0",
    "uuid": "^9.0.0",
    "xml2js": "^0.4.23",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^14.0.0",
    "@angular-eslint/builder": "~13.0.1",
    "@angular-eslint/eslint-plugin": "~13.0.1",
    "@angular-eslint/eslint-plugin-template": "~13.0.1",
    "@angular-eslint/template-parser": "~13.0.1",
    "@angular/cli": "^14.0.0",
    "@angular/compiler": "^14.0.0",
    "@angular/compiler-cli": "^14.0.0",
    "@angular/language-service": "^14.0.0",
    "@capacitor/cli": "^5.0.0",
    "@ionic/angular-toolkit": "^6.0.0",
    "@types/jasmine": "~3.6.0",
    "@types/jasminewd2": "~2.0.3",
    "@types/lodash.deburr": "^4.1.7",
    "@types/node": "^12.11.1",
    "@types/signature_pad": "^2.3.2",
    "@types/xml2js": "^0.4.11",
    "@typescript-eslint/eslint-plugin": "5.3.0",
    "@typescript-eslint/parser": "5.3.0",
    "eslint": "^7.6.0",
    "eslint-plugin-import": "2.22.1",
    "eslint-plugin-jsdoc": "30.7.6",
    "eslint-plugin-prefer-arrow": "1.2.2",
    "jasmine-core": "~3.8.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~6.3.2",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-coverage-istanbul-reporter": "~3.0.2",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "protractor": "~7.0.0",
    "ts-node": "~8.3.0",
    "tsickle": "^0.46.3",
    "typescript": "~4.7.3"
  },
  "description": "Mobile version of Flowww management software.",
  "prettier": {
    "singleQuote": true,
    "tabWidth": 4
  }
}
```

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by Ionitron** [comment]
It looks like this issue didn't get the information it needed, so I'll close it for now. If I made a mistake, sorry! I am just a bot.

Have a great day!
Ionitron 💙

**Comment by alexgerardojacinto** [comment]
@AntoniGadea we have released [@capacitor/camera@6.1.2](https://github.com/ionic-team/capacitor-plugins/releases/tag/%40capacitor%2Fcamera%406.1.2) which fixes the issue.
